test/distributed/org/apache/cassandra/distributed/impl/InstanceConfig.java

```diff
@@ -67,14 +69,15 @@ private InstanceConfig(int num,
                            String commitlog_directory,
                            String hints_directory,
                            String cdc_raw_directory,
-                           String initial_token,
+                           Collection<String> initial_token,
                            int storage_port,
                            int native_transport_port)
     {
         this.num = num;
         this.networkTopology = networkTopology;
         this.hostId = new UUID(0x4000L, (1L << 63) | num); // deterministic hostId for simulator
-        this    .set("num_tokens", 1)
+        this    .set("num_tokens", initial_token.size())
```

Comment:
Have we considered using public static constants for this, or are they ingrained enough in our collective culture that we're fine just using these param names raw in code + config?

We also reference num_tokens raw in ClusterUtils on getTokenCount (and a variety of other places). Given these are pretty calcified by virtue of being in the .yaml and thus part of our public API contract, probably fine just leaving things. Was just an observation while reading through this (lots of raw strings for params floating around here)

Reply:
`public static constants for this` do you mean `num_tokens`?

We could create a constants class, but rather not in this patch as this code base uses magic constants a lot already.

Reply:
Definitely. Just wanted to check my understanding; probably worth making a note to make a follow up ticket to try and eliminate some of these magic constants here.